### PR TITLE
docs: update links to docs site

### PR DESCRIPTION
**Pull request by stefanprobst**
this updates some links to the docs site to their new location

### CODE REVIEW

File: examples/with-google-analytics-amp/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+This example shows how to use [Next.js](https://github.com/vercel/next.js) along with [Google Analytics](https://developers.google.com/analytics/devguides/collection/gtagjs/) in conjunction with [AMP](https://nextjs.org/docs/advanced-features/amp-support/introduction). A custom [\_document](https://github.com/vercel/next.js/#custom-document) is used to inject [tracking snippet](https://developers.google.com/analytics/devguides/collection/gtagjs/) and track [pageviews](https://developers.google.com/analytics/devguides/collection/gtagjs/pages) and [event](https://developers.google.com/analytics/devguides/collection/gtagjs/events). There are two separate initializations of the Google Analytics tracking code; one for AMP and one for non-AMP pages.
```
Comment (chibicode) [inline examples/with-google-analytics-amp/README.md:3]:
```suggestion
This example shows how to use [Next.js](https://github.com/vercel/next.js) along with [Google Analytics](https://developers.google.com/analytics/devguides/collection/gtagjs/) in conjunction with [AMP](https://nextjs.org/docs/advanced-features/amp-support/introduction). A custom [\_document](https://nextjs.org/docs/advanced-features/custom-document) is used to inject [tracking snippet](https://developers.google.com/analytics/devguides/collection/gtagjs/) and track [pageviews](https://developers.google.com/analytics/devguides/collection/gtagjs/pages) and [event](https://developers.google.com/analytics/devguides/collection/gtagjs/events). There are two separate initializations of the Google Analytics tracking code; one for AMP and one for non-AMP pages.
```

**Review by chibicode — APPROVED** [review]
Checked that all the links work. Thanks!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| buildDuration | 12.3s | 12.5s | ⚠️ +217ms |
| nodeModulesSize | 67 MB | 67 MB | ⚠️ +52 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.093 | 2.078 | -0.02 |
| / avg req/sec | 1194.39 | 1202.95 | ⚠️ +8.56 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.191 | 1.234 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 2099.88 | 2025.39 | -74.49 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4dd5.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.html gzip | 953 B | 953 B | ✓ |
| link.html gzip | 960 B | 960 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| buildDuration | 13.1s | 13.5s | ⚠️ +395ms |
| nodeModulesSize | 67 MB | 67 MB | ⚠️ +52 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4dd5.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 906 kB | 906 kB | ✓ |
| withRouter.js | 906 kB | 906 kB | ✓ |
| Overall change | 4.48 MB | 4.48 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| buildDuration | 13.9s | 14s | ⚠️ +125ms |
| nodeModulesSize | 67 MB | 67 MB | ⚠️ +52 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.414 | 2.438 | ⚠️ +0.02 |
| / avg req/sec | 1035.55 | 1025.26 | -10.29 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.58 | 1.657 | ⚠️ +0.08 |
| /error-in-render avg req/sec | 1581.91 | 1508.72 | -73.19 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4dd5.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.html gzip | 953 B | 953 B | ✓ |
| link.html gzip | 960 B | 960 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| buildDuration | 15s | 15s | ⚠️ +48ms |
| nodeModulesSize | 67 MB | 67 MB | ⚠️ +52 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4dd5.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 906 kB | 906 kB | ✓ |
| withRouter.js | 906 kB | 906 kB | ✓ |
| Overall change | 4.48 MB | 4.48 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.3s | ⚠️ +110ms |
| nodeModulesSize | 67 MB | 67 MB | ⚠️ +52 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.052 | 2.08 | ⚠️ +0.03 |
| / avg req/sec | 1218.12 | 1201.65 | -16.47 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.214 | 1.222 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2059.49 | 2045.22 | -14.27 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4dd5.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.html gzip | 953 B | 953 B | ✓ |
| link.html gzip | 960 B | 960 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| buildDuration | 13.2s | 13.1s | -102ms |
| nodeModulesSize | 67 MB | 67 MB | ⚠️ +52 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..4dd5.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| _error.js gzip | 3.37 kB | 3.37 kB | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.05 kB | 2.05 kB | ✓ |
| hooks.js gzip | 881 B | 881 B | ✓ |
| _app.js gzip | 1.26 kB | 1.26 kB | ✓ |
| Overall change | 8.34 kB | 8.34 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| index.module.js gzip | 223 B | 223 B | ✓ |
| routerDirect..dule.js gzip | 281 B | 281 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| _error.module.js gzip | 2.21 kB | 2.21 kB | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| _app.module.js gzip | 604 B | 604 B | ✓ |
| Overall change | 5.49 kB | 5.49 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _buildManifest.js gzip | 270 B | 270 B | ✓ |
| _buildManife..dule.js gzip | 274 B | 274 B | ✓ |
| Overall change | 544 B | 544 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | stefanprobst/next.js docs/invalid-href-useful-link | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 906 kB | 906 kB | ✓ |
| withRouter.js | 906 kB | 906 kB | ✓ |
| Overall change | 4.48 MB | 4.48 MB | ✓ |

</details>

</details>

**Review by timneutkens — APPROVED** [review]

**Pull request merged**